index.js

```diff
@@ -521,6 +541,12 @@
     return typeof other === typeof this && other.valueOf() === this.valueOf();
   }
 
+  //  Boolean$prototype$lte :: Boolean ~> Boolean -> Boolean
+  function Boolean$prototype$lte(other) {
+    return typeof other === typeof this &&
+      (this.valueOf() ? other.valueOf() : true);
```

Comment:
I find it easier to comprehend. It will evaluate `true` if `x` is `false` (regardless of `y`) or if `y` is `true` (regardless of `x`). Somehow this is closer to my mental model than the equivalent ternary expression.